pkg/flowaggregator/flowaggregator.go

```diff
@@ -115,6 +117,10 @@ var (
 		"reversePacketDeltaCountFromDestinationNode",
 		"reversePacketTotalCountFromDestinationNode",
 	}
+	antreaLablesElementList = []string{
```

Comment:
why do we need a separate list here?

Reply:
Because `sourcePodLabels` and `destinationPodLabels` fields are inserted at the end of each record, so I add a separate list here to put them at the end of template record.